lib/index.js

```diff
@@ -3,7 +3,7 @@
 const ClientVersion = require('../package.json').version;
 const Resources     = require('./resources');
 
-const LOB_HOST = 'https://api.lob.com/v1/';
+const LOB_HOST = process.env.LOB_HOST || 'https://api.lob.com/v1/';
```

Comment:
I add this so I could test in staging by passing `LOB_HOST` in when running `npm test`